examples/logconcave/CMakeLists.txt

```diff
@@ -145,6 +142,8 @@ else ()
   TARGET_LINK_LIBRARIES(simple_sde ${LP_SOLVE} ${BLAS}  "-L${MKLROOT}/lib/intel64 -Wl,--no-as-needed -lmkl_intel_ilp64 -lmkl_gnu_thread -lmkl_core -lgomp -lpthread -lm -ldl")
```

Comment:
please create an MKL linking variable as in the tests that contain all of these flags